src/sentry/api/serializers/models/groupseen.py

```diff
@@ -1,11 +1,17 @@
-from sentry.api.serializers import Serializer, register, serialize
+from sentry.api.serializers import Serializer, register
 from sentry.models import GroupSeen
+from sentry.services.hybrid_cloud.user import user_service
 
 
 @register(GroupSeen)
 class GroupSeenSerializer(Serializer):
     def get_attrs(self, item_list, user):
-        user_map = {d["id"]: d for d in serialize({i.user for i in item_list}, user)}
+        serialized_users = user_service.serialize_many(
+            filter=dict(user_ids=[i.user_id for i in item_list]), as_user=user
+        )
+        user_map = {}
+        for serialized in serialized_users:
+            user_map[serialized["id"]] = serialized
```

Comment:
Probably better that we rely on doing this explicitly in the serializer rather hoping the caller remembers to use `select_related`, good change